Python/Product/TestAdapter.Executor/Pytest/PytestExtensions.cs

```diff
@@ -22,37 +22,44 @@
 namespace Microsoft.PythonTools.TestAdapter.Pytest {
     static class PyTestExtensions {
 
-        public static TestCase ToVsTestCase(this PytestTest test, string projectHome, Dictionary<string, PytestParent> parentMap) {
-            if (String.IsNullOrEmpty(projectHome)) {
-                throw new ArgumentException(nameof(projectHome));
+
```

Comment:
move source and line parsing to its own function.

its expected that pytest discovery returns sources that are relative to project/workspace. i should add a comment